chapters/classes.tex

```diff
@@ -217,7 +217,9 @@ \subsubsection{Prefix Rules}\label{prefix-rules}
 
 A \firstuse{prefix} is property of an element of a class definition which can be present or not be present, e.g., \lstinline!final!, \lstinline!public!, \lstinline!flow!.
 
-Variables declared with the \lstinline!flow! or the \lstinline!stream! type prefix shall be a subtype of \lstinline!Real!.
+Variables declared with the \lstinline!stream! type prefix shall be a subtype of \lstinline!Real!.
+Variables declared with the \lstinline!flow! type prefix shall be a subtype of \lstinline!Real! or an operator record definining certain operators, see \cref{generation-of-connection-equations}.
```

Comment:
The sentence on line 224 allows streams and flows to be applied to record and connector components. (Sorry, GitHub does not allow me to add the comment to that line.) It does not mention operator record explicitly. Is it assumed that record means operator record? Either way, there is still a contradiction between lines 220-221 and 224.

Reply:
I agree it is a mess, and I have now tried to improve it.
Some of the problem was that I missed some text on ridiculously long lines.

Note that the text for applying stream, flow, input, output the text is somewhat more complicated; since even if we don't allow 'stream' on a structured component we still want to keep the restriction that if you use 'input' on a structured component there are no 'stream' members in it.

That section could still be cleaned up/restructured further, but I think that can wait.